README.md

```diff
@@ -162,8 +162,7 @@ Download the latest `preprocess` zip source package, unzip it, and run `python
 setup.py install`:
 ```
     unzip master.zip
-    cd preprocess
-    python setup.py install
+    pip install preprocess-master/
```

Comment:
I think it would be better to simply do it all in one command:
``` console
pip install https://github.com/doconce/preprocess/archive/master.zip
```
The text would also need to be updated.